modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -793,10 +794,28 @@ exports.modifyUserTag = function(req, res) {
       message += (joining ? 'Joined' : 'Left');
       message += ' ' + ((tag && tag.tribe) ? 'tribe' : 'tag') + '.';
 
-      return res.send({
-        message: message,
-        tag: pickedTag,
-        user: user
+      statService.stat({
+        namespace: 'tag',
+        counts: {
+          count: joining ? 1 : -1
```

Comment:
@mrkvon can I use this `counts` like this?

(feel free to suggest any better format)